variables.tf

```
@@ -56,6 +56,42 @@ variable "public_lb_target_group_name" {
   default     = ""
 }
 
+variable "public_lb_tls_listener" {
+  type        = string
+  description = "Name of the public ALB TLS listener"
+  default     = ""
+}
+
+variable "public_lb_tls_listener_rule_token" {
+  type        = string
+  description = "Name of the public ALB TLS listener rule with token authorization"
+  default     = ""
+}
+
+variable "public_lb_tls_listener_rule_whitelist" {
+  type        = string
+  description = "Name of the public ALB TLS listener rule with ip authorization"
+  default     = ""
+}
+
+variable "public_lb_http_listener" {
+  type        = string
+  description = "Name of the public ALB HTTP listener"
+  default     = ""
+}
+
+variable "public_lb_http_listener_rule_token" {
+  type        = string
+  description = "Name of the public ALB HTTP listener rule with token authorization"
+  default     = ""
+}
+
+variable "public_lb_http_listener_rule_whitelist" {
+  type        = string
+  description = "Name of the public ALB HTTP listener rule with ip authorization"
+  default     = ""
+}
+
```

It would be more understandable that way imho.

```suggestion
variable "public_lb_tls_listener_name" {
  type        = string
  description = "Name of the public ALB TLS listener"
  default     = ""
}

variable "public_lb_tls_listener_rule_token_name" {
  type        = string
  description = "Name of the public ALB TLS listener rule with token authorization"
  default     = ""
}

variable "public_lb_tls_listener_rule_whitelist_name" {
  type        = string
  description = "Name of the public ALB TLS listener rule with ip authorization"
  default     = ""
}

variable "public_lb_http_listener_name" {
  type        = string
  description = "Name of the public ALB HTTP listener"
  default     = ""
}

variable "public_lb_http_listener_rule_token_name" {
  type        = string
  description = "Name of the public ALB HTTP listener rule with token authorization"
  default     = ""
}

variable "public_lb_http_listener_rule_whitelist_name" {
  type        = string
  description = "Name of the public ALB HTTP listener rule with ip authorization"
  default     = ""
}

```